server/src/routes/api/v1/parkReviewsRouter.js

```diff
@@ -34,4 +34,36 @@ parkReviewsRouter.post("/", async (req, res) => {
   }
 });
 
+parkReviewsRouter.post("/:reviewId", async (req, res) => {
+  try {
+    const reviewId = req.params.reviewId;
+    const parkId = req.params.parkId;
+    await Review.query().deleteById(reviewId);
+    const park = await Park.query().findById(parkId);
+    const serializedPark = await ParkSerializer.showDetails(park);
+    return res.status(201).json({ park: serializedPark });
```

Comment:
then, if you need the average rating, you can re-calculate it as necessary. this endpoint shouldnt return a park, but instead either all of the reviews sans the deleted one